src/charm.py

```diff
@@ -211,6 +211,10 @@ def _make_env_config(self, domain="svc.cluster.local") -> dict:
                 backend_site_name = urlparse(backend).hostname
             site = config["site"]
 
+        cache_all_configs = ''
+        if config['cache_all'] == False:
+            cache_all_configs = "proxy_ignore_headers Cache-Control Expires;"
```

Comment:
Do you think others should be ignored too?
“X-Accel-Expires”, “Expires”, “Cache-Control”, “Set-Cookie”, and “Vary” set the parameters of response caching

Reply:
Tbh idk, I only maintained what was on the original MP and made it work. Does anyone have an opinion/recommendation on this matter?

Reply:
I'd keep for this PR if it works